typings/index.d.ts

```diff
@@ -566,6 +566,9 @@ export class ClientApplication extends Application {
   public commands: ApplicationCommandManager;
   public cover: string | null;
   public flags: Readonly<ApplicationFlags>;
+  public tags?: [string, string?, string?, string?, string?];
```

Comment:
Since `ClientApplication#tags` is now required:
```suggestion
  public tags: [string?, string?, string?, string?, string?];
```

Reply:
The first `string` in the tuple is still required, since a if an app has tags, it must have at least one.

Reply:
But since you made the array default to an empty array, that's not longer the case.

Reply:
You’re defaulting the tags to an empty array though, so that can be missing